src/Mado/QueryBundle/Interfaces/CustomFilter.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Mado\QueryBundle\Interfaces;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Mado\QueryBundle\Component\Meta\GraphWalker;
+use Mado\QueryBundle\Services\IdsChecker;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+interface CustomFilter
+{
+    public function __construct(
+        EntityManagerInterface $manager,
+        GraphWalker $walter,
+        RequestStack $requestStack,
+        IdsChecker $idsChecker,
+        LoggerInterface $logger
+    );
+
+    public function setUser(AdditionalFilterable $user);
+
+    public function allItemsTo(string $entity);
+
+    public function getPathTo(string $fullyqualifiedClassName);
```

Comment:
change this:
fullyqualifiedClassName

to this:
fullyQualifiedClassName

Reply:
👍